src/store/BouquetStore.js

```diff
@@ -0,0 +1,56 @@
+import { defineStore } from "pinia"
+import TopicsAPI from "../services/api/resources/TopicsAPI"
+
+const topicsAPI = new TopicsAPI()
+
+export const useBouquetStore = defineStore("bouquet", {
+  state: () => ({
+    data: [],
+  }),
+  actions: {
+    /**
+     * Filter a list of bouquets related to Ecospheres
+     *
+     * @param {Array} bouquets
+     * @returns {Array}
+     */
+    filter (bouquets) {
+      return bouquets.filter(bouquet => bouquet.extras.is_ecospheres || bouquet.tags.includes("ecospheres"))
+    },
+    /**
+     * Load Ecospheres related bouquets from API
+     *
+     * @returns {Array<object>}
+     */
+    async loadBouquets () {
```

Comment:
So French! But `loadBunches`... We may have to find another term 😅
(not a change request, just me wondering how i18n is going to work when we need it)

Reply:
Yep :p Bouquet is legit though, I checked 🇬🇧